modules/react-native-live-camera-filters/package.json

```diff
@@ -5,15 +5,15 @@
   "private": true,
   "main": "index.js",
   "x-dependencies": {
-    "@react-native-community/cameraroll": "^4.1.2",
+    "@react-native-camera-roll/camera-roll": "^5.7.2",
     "react-native-vector-icons": "^9.2.0",
-    "react-native-filter-camera": "0.1.18"
+    "react-native-filter-camera": "github:saad-abid-crowdbotics/react-native-filter-camera"
   },
   "author": "Crowdbotics",
   "license": "ISC",
   "dependencies": {
-    "@react-native-community/cameraroll": "^4.1.2",
+    "@react-native-camera-roll/camera-roll": "^5.7.2",
     "react-native-vector-icons": "^9.2.0",
-    "react-native-filter-camera": "0.1.18"
+    "react-native-filter-camera": "github:saad-abid-crowdbotics/react-native-filter-camera"
```

Comment:
Why is this needed?

Reply:
@saad-abid-crowdbotics can comment on this but this was related to new scaffold 0.71 support there was some issue so he had to sort of monkeypatch this

Reply:
This module uses the "react-native-filter-camera" library to apply filters. The "react-native-filter-camera" library relies on another native android library called "com.otaliastudios:cameraview" v2.6.2 for Android. However, this version of the library is not compatible with the New Scaffold version 0.71.7. It requires v2.7.2. To address this issue, There were two solutions to this either I forked the library or change the version in "node_modules/react-native-filter-camera/android/build.gradle"